GVFS/GVFS.Common/GVFSLock.cs

```diff
@@ -319,7 +331,11 @@ public void AddStatsToTelemetry(EventMetadata metadata)
                 metadata.Add("DurationMS", this.lockAcquiredTime.ElapsedMilliseconds);
                 metadata.Add("LockHeldExternallyMS", this.lockHeldExternallyTimeMs);
                 metadata.Add("ParseGitIndexMS", this.parseGitIndexTimeMs);
+                metadata.Add("UpdatePlaceholdersTotalMS", this.placeholderTotalUpdateTimeMs);
                 metadata.Add("UpdatePlaceholdersMS", this.placeholderUpdateTimeMs);

```

Comment:
Should `"UpdatePlaceholdersMS"` be changed to something like `"UpdateExistingPlaceholdersMS"` to avoid any possible confusion with its old meaning (of total time)?